doc/Example of Cross Compiling the Azure IoT SDK for Python.md

```diff
@@ -0,0 +1,230 @@
+```
+You will need to create two files in this directory. With the editor of your choice first create _toolchain-mlinux.cmake_. Into this file enter the following:
+```cmake
+INCLUDE (CMakeForceCompiler)
+SET (CMAKE_SYSTEM_NAME Linux)     # this one is important
+SET (CMAKE_SYSTEM_VERSION 1)      # this one not so much
+
+# Set up some paths for the build
+SET (CMAKE_SYSROOT $ENV{ML_SYSROOT})
+SET (CMAKE_LIBRARY_PATH "$ENV{ML_SYSROOT}/usr/lib")
+SET (CMAKE_PREFIX_PATH "$ENV{ML_SYSROOT}/usr")
+SET (Boost_INCLUDE_DIR "$ENV{BOOST_ROOT}/include")
+SET (Boost_LIBRARY_DIR_RELEASE "$ENV{BOOST\_ROOT}/lib")
+
+# Uncomment for a little debug info
+#message("Boost_INCLUDE_DIR ${Boost_INCLUDE_DIR}")
+#message("Boost_LIBRARY_DIR_RELEASE ${Boost_LIBRARY_DIR_RELEASE}")
+
+# Set up the compilers and the flags
+SET (CMAKE_C_COMPILER $ENV{ML_HOSTTOOLS}/arm-mlinux-linux-gnueabi/arm-mlinux-linux-gnueabi-gcc)
+SET (CMAKE_CXX_COMPILER $ENV{ML_HOSTTOOLS}/arm-mlinux-linux-gnueabi/arm-mlinux-linux-gnueabi-g++)
+SET (CMAKE_C_FLAGS "-march=armv5te -marm -mthumb-interwork -mtune=arm926ej-s -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed")
+SET (CMAKE_CXX_FLAGS "-march=armv5te -marm -mthumb-interwork -mtune=arm926ej-s -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed")
+
+# This is the file system root of the target
+SET (CMAKE_FIND_ROOT_PATH $ENV{ML_SYSROOT})
+
+# Search for programs in the build host directories
+SET (CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+
+# For libraries and headers in the target directories
+SET (CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
+
+SET (CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+```
+<div style="text-align: center; font-size: smaller">Figure 2: CMake toolchain file</div>
+
+Create a second file called _mlinuxenv.sh_ and into it enter the following:
+```bash
+export ML_TOOLCHAINROOT=${HOME}/mlinux/3.3.6/sysroots
+export ML_HOSTTOOLS=${ML_TOOLCHAINROOT}/x86\_64-mlinux-linux/usr/bin
+export ML_SYSROOT=${ML_TOOLCHAINROOT}/arm926ejste-mlinux-linux-gnueabi
+export BOOST_ROOT=${HOME}/boostmlinux-1-64-0
+export PATH=${ML_HOSTTOOLS}/bin:${ML_HOSTTOOLS}/bin/arm-mlinux-linux-gnueabi:$PATH
+
+cmake -DCMAKE_INSTALL_PREFIX=~/mlinux-out -DCMAKE_TOOLCHAIN_FILE=../build_all/linux/toolchain-mlinux.cmake ..
+```
+<div style="text-align: center; font-size: smaller">Figure 3: Build Generator Script</div>
+
+Having created the two files, we are now ready to build the SDK. Enter the following commands:
+```
+chmod a+x mlinuxenv.sh
+cd ../../cmake
+../build_all/linux/mlinuxenv.sh
+make install
+```
+This will build the SDK and create the Python library. To check that the binary Python library has been built correctly enter:
+```
+file ~/mlinux-out/lib/*.so
+```
+This should result in two lines of output, one for _iothub\_client.so_ and one for _iothub\_service\_client.so_. The details should say that they are _ELF 32-bit LSB shared object, ARM, EABI5 version 1 (SYSV), dynamically linked_ which is the appropriate architecture for your target machine.
+
+## Conclusion
```

Comment:
Consider: to make it complete please add here the steps to copy samples here from the repo and run it with command line parameters.